src/e3/spdx.py

```diff
@@ -957,12 +962,41 @@ def add_section(section: str) -> None:
         add_section("Creation Info")
         output += self.creation_info.to_tagvalue()
 
+        # Keep track of the packages which should appear first.
+        pkg_ids: list[SPDXID] = []
         add_section("Relationships")
-        for rel in self.relationships:
-            output.append(rel.to_tagvalue())
-        for pkg in self.packages.values():
-            add_section("Package")
-            output += pkg.to_tagvalue()
+        relas: list[str] = []
+        rel_ix: int = 0
+        for rel in sorted(
+            self.relationships, key=lambda rela: rela.related_spdx_element
+        ):
+            if (
+                rel.spdx_element_id == self.spdx_id
+                and rel.relationship_type == RelationshipType.DESCRIBES
+            ):
+                # Make the described packages appear first in list.
+                relas.insert(rel_ix, rel.to_tagvalue())
+                rel_ix += 1
+                # Keep track of those leading packages IDs.
+                pkg_ids.append(rel.related_spdx_element)
+            else:
+                relas.append(rel.to_tagvalue())
+
+        output += relas
+
+        packages: list[str] = ["", ""]
+
+        for pkg in sorted(self.packages.values(), key=lambda package: package.name):
+            print(f"Package ID: {pkg.spdx_id}, Document ID: {self.spdx_id}")
```

Comment:
Is this print statement expected?

Reply:
:'( In the middle of the stage :'( Sorry ...

Reply:
Sorry, due to the presence of the `platform` package in `e3`, it became impossible to debug `e3` with PyCharm. So I added logs.\
I have to dig the configuration to make it possible (again) to debug `e3` ...